include/seqan3/alignment/configuration/utility.hpp

```diff
@@ -165,8 +167,9 @@ inline constexpr align_cfg::id align_config_type_to_id_v = align_config_type_to_
 inline constexpr std::array<std::array<bool, static_cast<uint8_t>(align_cfg::id::SIZE)>,
                             static_cast<uint8_t>(align_cfg::id::SIZE)> align_config_validation_matrix =
 {
-    // gap
-    { false }  // gap
+    // gap   band
+    { false, true,   // gap
```

Comment:
Yes, it compiles. `align_cfg::id::SIZE` is a member of the `enum struct id` some lines above and it denotes the number of contained ids, so value 2 is correct. 

The matrix here is indeed a fixed 2x2 array. It can be initialised with one bracket as described on this page: https://en.cppreference.com/w/c/language/array_initialization#Nested_arrays